R/get_ssm_by_patients.R

```diff
@@ -3,58 +3,60 @@
 #' @description Get MAF-format data frame for more than one patient.
 #'
 #' @details This function returns variants from a set of patients.
-#' This function internally calls [GAMBLR.data::get_ssm_by_samples]. 
-#' Thus, the main contents of this function is to wrangle the provided patient IDs, 
+#' This function internally calls [GAMBLR.data::get_ssm_by_samples].
+#' Thus, the main contents of this function is to wrangle the provided patient IDs,
 #' so that the corresponding sample IDs can be provided to the internal call of `get_ssm_by_samples`.
-#' This function expects either a vector of patient IDs (`these_patients_ids`) 
+#' This function expects either a vector of patient IDs (`these_patients_ids`)
 #' or an already subset metadata table (`these_samples_metadata`).
 #'
-#' @param these_patient_ids A vector of patient IDs that you want results for. 
+#' @param these_patient_ids A vector of patient IDs that you want results for.
 #' The user can also use a metadata table that has been subset to the patient IDs of interest (see `these_samples_metadata`).
-#' @param these_samples_metadata A metadata subset to contain the rows corresponding to the patients of interest. 
+#' @param these_samples_metadata A metadata subset to contain the rows corresponding to the patients of interest.
 #' If the vector of patient IDs is missing (`these_patient_ids`), this function will default to all patient IDs in the metadata table given to this parameter.
 #' @param projection Obtain variants projected to this reference (one of grch37 or hg38). Default is grch37.
 #' @param this_seq_type The seq type you want results for. Default is "genome".
+#' @param tool_name Optionally specify which tool to report variant from. The default is slms-3, also supports "publication" to return the exact variants as reported in the original papers.
```

Comment:
For some functions this argument is called `tool_name`, for other ones it's called `mode`. Should they be harmonized? Or is there a reason for the different names?

Reply:
The names of the arguments for these functions are the same as the names of the arguments in the parental functions in GAMBLR.results. I agree they need to be harmonized eventually but that would have to be separate effort focused specifically on the argument names.